frontend/lib/package.json

```diff
@@ -59,6 +59,7 @@
     "color2k": "^2.0.2",
     "d3": "^7.8.2",
     "d3-graphviz": "^2.6.1",
+    "date-fns": "^2.6.0",
```

Comment:
**note:** This was previously an existing transitive dependency. I added it here to make it explicit / ensure the eslint rule about dependencies is happy.